hub/core/chunk_engine.py

```diff
@@ -322,7 +322,8 @@ def append(self, sample: SampleValue):
         else:
```

Comment:
@mccrearyd are you sure that `_append_bytes` doesn't do any insertion that would trigger `maybe_flush`?

Reply:
it is covered

Reply:
I think what @benchislett meant is that _append_bytes already writes to cache and cache setitem would anyway trigger maybe_flush and these new changes don't really help much.

Reply:
ohhh, ok closing